projects/js-packages/components/components/dialog/index.tsx

```diff
@@ -18,9 +24,9 @@ import styles from './style.module.scss';
  * @param {React.Component} props.primary   - Primary content.
  * @param {React.Component} props.secondary - Secondary content.
  * @param {boolean} props.split			    - Split the sections.
- * @returns {object}                          Dialog react component.
+ * @returns {React.FunctionComponent}         Dialog react component.
```

Comment:
`FunctionComponent`  means which can be instantiated i.e. which can be used as a component.
```suggestion
 * @returns {React.ReactNode}         Rendered dialog
```